docs/Tutorials/Tutorial-design.md

```diff
@@ -0,0 +1,38 @@
+# Tutorial Design
+
+The purpose of a hack week is to expose participants to a broad range of data science tools and methods. With this in mind, we suggest the following guidelines when developing a tutorial:
+
+* Tutorials should be designed to address broad challenges in the data science community being served by the event. Speaking with community groups, reading blogs and attending workshops are good ways to connect with the most urgent data science challenges in a particular community. For example, the ICESat-2 hack week tutorial content was largely informed by members of the satellite's science team who were directly involved in releasing the data to the research community. 
+* Based on the needs of the community, and in consultation with other event planners, build an outline of desired learning outcomes.  
+* Consider how you would like to balance teaching on core concepts versus the technical aspects of operating a specific tool or library. Be mindful of spending too much time on a tool you might have just discovered that represents the latest and greatest, but might not be something the community decides to adopt in the long term. 
+* Include content that provides an initial picture of what might be possible when applying a particular data science tool. Save detailed explanations on a narrow topic for a later breakout session.
+* Start with simple explanations that attend to people who are seeing this content for the first time, AND keep advanced participants engaged by inviting them to assist others, or to explore more advanced concepts through individual study.
+* Make your tutorials interactive through live coding challenges, working through a Juypter Notebook together, or interactive discussions. Leave some space when you are not talking for people to work through examples and ask questions. 
+* take a bit of time to review tutorials offered at our previous hackweeks. [Here](https://www.youtube.com/watch?v=VMJ1ZO48cwU) is an example of a particularly well-crafted tutorial by Catherine Kuhn at the 2018 Geo Hack Week.
+
+## Tutorial Timeline
+
+It is a good idea to prepare your tutorials well in advance of the event so that they can be reviewed by other tutorial developers and members of the community. In some cases early preparation is required so that shared computing resources can be tested and configured to contain the libraries and datasets necessary to run the tutorials. Early tutorial preparation also cretes an opportunity for exploring the overall flow of tutorial ideas and concepts across the event. 

```

Comment:
Out of curiosity: how do you do this for GeoHackWeek? I would love to have tutorials ready early, but everyone I know (including me) generally finishes their tutorials the day before (or the day of). I once taught at a summer school, where the organizers said six months in advance "we need your finished tutorials in two weeks", and I was not super impressed by that, either.

Reply:
We ask for things 1 month in advance so we can test everything on JupyterHub, but that is aspirational...usually it is the day before!